examples/Cargo.toml

```diff
@@ -29,8 +31,13 @@ gtkvideooverlay-x11 = ["gtkvideooverlay"]
 gtkvideooverlay-quartz = ["gtkvideooverlay"]
 tokio = ["gstreamer/futures", "futures", "tokio-core"]
 gst-rtsp-server = ["gstreamer-rtsp-server"]
+gst-rtsp-server-record = ["gstreamer-rtsp-server", "gstreamer-rtsp", "gio"]
 default-features = []
 v1_10 = ["gstreamer/v1_10"]
 
 [badges]
 travis-ci = { repository = "sdroege/gstreamer-rs", branch = "master" }
+
+[[bin]]
+name = "rtsp-server-record"
+required-features = ["gst-rtsp-server-record"]
```

Comment:
Hm, does this work as expected? In that case can you add a similar thing for all the other examples in a separate commit? And potentially clean up the code to not be ifdef disaster

Reply:
Done